content/ja/handbook/engineering/ai/custom-models/index.md

```
@@ -0,0 +1,227 @@
+
+### プランニングボード
+
+[Planning Board](https://gitlab.com/groups/gitlab-org/-/boards/7762631?milestone_title=17.7&label_name[]=group%3A%3Acustom%20models) は各マイルストーンに対して PM によって作成され、カテゴリ別にキュレーションされた Issue のリストです。プランニングボードは Issue で過剰に埋まることがあり、超過分はプランニングコール中に次マイルストーンまたは Next 1-3 Milestones ボードに移動されます。
+
+PM は Issue に `~workflow::planning breakdown` をマークします。これは EM に対して、Issue 説明が明確で開発の準備ができていることを確認するため、エンジニアにレビューを依頼するよう合図するものです。エンジニアはその後ウェイトを割り当て、`~workflow::ready for development` ラベルを適用します。
+
+### Issue リファインメントの自動化
+
+毎週、Issue リファインメントを支援するために [Custom Models プロジェクト](https://gitlab.com/gitlab-org/ai-powered/custom-models/custom-models) 内に新しい Issue が作成されます。
+
+エンジニアは毎週 Issue をリファインし、タスクをレビューして複雑度を見積もり、開発に向けて準備します。リファインメントプロセス中に、必要な作業を評価し、必要に応じて実装計画を追加し、[ウェイト](#issue-weights) を加えて、Issue を [開発準備完了](#ready-for-development-status) としてマークします。このプロセスにより、Issue が開発開始前にしっかり定義されていることを保証します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)